Library/Homebrew/tab.rb

```diff
@@ -31,7 +31,12 @@ def self.create(formula, compiler, stdlib)
       "source" => {
         "path" => formula.path.to_s,
         "tap" => formula.tap ? formula.tap.name : nil,
-        "spec" => formula.active_spec_sym.to_s
+        "spec" => formula.active_spec_sym.to_s,
+        "versions" => {
+          "stable" => formula.stable ? formula.stable.version.to_s : nil,
+          "devel" => formula.devel ? formula.devel.version.to_s : nil,
+          "head" => formula.head ? formula.head.version.to_s : nil
+        }
```

Comment:
:+1: Already implemented it locally. Haven't pushed the changes yet, but I will.